public/components/security-management-app.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { EuiTabbedContent, EuiTabbedContentTab } from '@elastic/eui';
+import { EuiPanel } from '@elastic/eui';
+import { RoleManagement } from './security-management/roles/role-management-app';
+import { AppMountContext } from '../../../../src/core/public';
+
+class SecurityManagementAppProps {
+  element!: HTMLElement;
+  appMountContext!: AppMountContext;
+  basePath!: string;
+}
+class SecurityManagementAppState {
+  selectedTab?: EuiTabbedContentTab;
+}
+
+class SecurityManagementApp extends Component<SecurityManagementAppProps, SecurityManagementAppState> {
+  private tabs: EuiTabbedContentTab[] = [
+    {
+      id: 'roles',
+      name: 'Roles',
+      content: (<RoleManagement/>)
+    },
+    {
+      id: 'action_groups',
+      name: 'Action Groups',
+      content: (<div>bbb</div>)
```

Comment:
is this a placeholder and to be implemented

Reply:
yes, I just explored how can a new UI plugin been registered using the new platform, the browser application is being developed by Yuxiang, he will remove my code like this with his implementation